src/app/books/books-routing.module.ts

```diff
@@ -1,12 +1,12 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
-import { BookDetailsContainer } from './containers/book-details/book-details.container';
-import { BooksListContainer } from './containers/books-list/books-list.container';
+import { BooksListContainer, BookDetailsContainer, BookCreateContainer } from './containers';
 
 const routes: Routes = [
   { path: '', component: BooksListContainer },
-  { path: 'books/:id', component: BookDetailsContainer },
+  { path: 'book-create', component: BookCreateContainer },
```

Comment:
name of path looks like strange. I suppose you already have the following url: `shop.com/books`. So, your path can be more pretty. Lets rename to `create` since you have `/books`.